Documents/TechnicalSpecifications/technicalSpecifications.md

```diff
@@ -38,15 +75,18 @@ In the team, two different types of computer are used:
 | Computer | OS | Number used |
 | -- | -- | -- |
 | MacBook Air M3 | MacOS | 4 |
-| ThinkBook 14 G4 | Windows 11 | 2 |
+| ThinkBook 14 G4 | Windows 11 | 3 |
+
+Thus, the game that we will create will work on Windows and MacOS operating systems only. 
 
 ## Programming language
 
-For this project, the Godot engine will be used. We will develop the game using its own programming language, GDscript.
+For this project, the Godot engine will be used. We will develop the game using its own programming language, GDscript. Those are used because it is easy to use, to maintain and for its portability.
 
 ## Conventions
 
 ### File tree and documents
+---
 
 Folders are written in "CamelCase"<br>
 Files are written in "pascalCase"
```

Comment:
Be careful you inverted both.

ThisIsPascaleCase and thisIsCamelCase

Reply:
done 👍